addons/hr_holidays/views/hr_leave_accrual_views.xml

```diff
@@ -184,8 +184,8 @@
                                         </div>
                                     </div>
                                     <t t-name="level_content">
-                                        <div class="o_hr_holidays_card">
-                                            <div class="content container" style="width: 560px;">
+                                        <div class="o_hr_holidays_card position-relative ms-4 px-3 py-2 border bg-view bg-opacity-100">
+                                            <div class="content container position-relative" style="width: 560px;">
```

Comment:
commit 771dc529383f66446aa307baada638a0c9ce192c
I think we should seize the opportunity of the refactoring the get rid of the inline-style in this file. Check other occurences as well 💪